src/components/ErrorBoundary/__tests__/ErrorBoundary-test.js

```diff
@@ -16,41 +16,43 @@ describe('components/ErrorBoundary', () => {
         wrapper.find(WrappedComponent).simulateError(wrappedError);
     };
 
-    describe('should render', () => {
+    describe('render()', () => {
```

Comment:
Most of the test assertions read a bit oddly with this and their other changes:

`components/ErrorBoundary render() the wrapped component when no error is thrown`
`components/ErrorBoundary render() should render default error component an error is thrown`

Reply:
I changed this to follow the existing pattern we use in EUA and elements

Reply:
That seems fine, but can we make sure that the sentences are consistent and make sense when read? See the examples above.